sphinx/source/docs/installation.rst

```diff
@@ -30,12 +30,20 @@ dependency on several compiled libraries.
 Ideally, you should have the following libraries installed:
 
 * NumPy
+* Pandas
 * Flask
 * Redis
+* Redis-py
+* Six
 * Requests
-* gevent
-* gevent-websocket
-* Pandas
+* Tornado >= 4.0
+* Werkzeug
+* Greenlet
+* Gevent
+* Gevent-websocket
+* PyZMQ
+* PyYaml
+* DateUtil
 
```

Comment:
I remember installing a previous version, then un-install it and setup.py develop there... but even if this work (not sure if could be a side effect) it is not the recommended way to proceed... what we need is just provide a conda requirement.txt, so people can do `conda create -n bokeh_env --file req.txt`